pkg/data/components/RemoveAttributeProcessor.yaml

```
@@ -0,0 +1,75 @@
+kind: RemoveAttributeProcessor
+name: Remove Attribute Processor
```

We dont need to include `processor` in the name displayed to the customer
```suggestion
name: Remove Attributes
```